__tests__/app.test.js

```diff
@@ -103,4 +105,39 @@ describe('App', () => {
         });
     });
   });
+
+  describe('/api/articles', () => {
+    test('GET:200 responds with a list of articles', () => {
+      return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles.length).toBe(13);
+        response.body.articles.forEach((article) => {
+          expect(typeof article.article_id).toBe('number');
+          expect(typeof article.title).toBe('string');
+          expect(typeof article.topic).toBe('string');
+          expect(typeof article.author).toBe('string');
+          expect(typeof article.created_at).toBe('string');
+          expect(typeof article.votes).toBe('number');
+          expect(typeof article.article_img_url).toBe('string');
+          expect(typeof article.comment_count).toBe('number');
+        });
+      });
+    });
+
+    describe('each article', () => {
+      test('should be sorted by date in descending order and should not have a body property', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+          expect(response.body.articles).toBeSortedBy('created_at', { descending: true });
+          response.body.articles.forEach((article) => {
+            expect(article).not.toHaveProperty('body')
```

Comment:
Having a sparate test for the order is great but the assertion for the exclusion of the body property would be better in it's own test too or maybe in the test above where you are looking at the shape of the articles